__tests__/data/engage.json

```diff
@@ -0,0 +1,590 @@
+[
+    {
+        "description": "[IDENTFY]: Create a new User with Metadata and lists from externalId",
+        "input": {
+            "message": {
+                "userId": "1",
+                "context": {
+                    "externalId": [
+                        {
+                            "type": "engageListId",
+                            "id": [
+                                "100c983ry8934hf3094yfh348gf1",
+                                "4r40hfio3rbfln"
+                            ]
+                        }
```

Comment:
ids should not come in an array. it should be broken as 

 "externalId": [
                        {
                            "type": "engageListId",
                            "id": [
                                "100c983ry8934hf3094yfh348gf1"
                            ]
                        },
                         {
                            "type": "engageListId",
                            "id": [
                                "4r40hfio3rbfln"
                            ]
                        }
                        ]

Reply:
Yeah I tried that but only the last one was coming to me. Like in the above case only the "4r40hfio3rbfln" would have come

Reply:
Okay